test/plugins/sensor_demo/custom_sensor/CustomRenderingSensor.hh

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_SENSORS_CustomRenderingSensor_HH_
+#define IGNITION_SENSORS_CustomRenderingSensor_HH_
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+#include <sdf/sdf.hh>
+
+#include <ignition/common/SuppressWarning.hh>
+
+#ifdef _WIN32
+#pragma warning(push)
+#pragma warning(disable: 4005)
+#pragma warning(disable: 4251)
+#endif
+#include <ignition/msgs.hh>
+#ifdef _WIN32
+#pragma warning(pop)
+#endif
+
+#ifdef _WIN32
+#pragma warning(push)
+#pragma warning(disable: 4251)
+#endif
+#include <ignition/rendering/Camera.hh>
+#ifdef _WIN32
+#pragma warning(pop)
+#endif
+
+#include "ignition/sensors/camera/Export.hh"
+#include "ignition/sensors/config.hh"
+#include "ignition/sensors/Export.hh"
+#include "ignition/sensors/RenderingSensor.hh"
+
+namespace ignition
+{
+  namespace sensors
+  {
+    // Inline bracket to help doxygen filtering.
+    inline namespace IGNITION_SENSORS_VERSION_NAMESPACE {
+    //
+    /// \brief forward declarations
+    class CustomRenderingSensorPrivate;
+
+    /// \brief Custom Camera Sensor Class
+    class IGNITION_SENSORS_CAMERA_VISIBLE CustomRenderingSensor : public RenderingSensor
+    {
+      /// \brief constructor
+      public: CustomRenderingSensor();
+
+      /// \brief destructor
+      public: virtual ~CustomRenderingSensor();
+
+      /// \brief Load the sensor based on data from an sdf::Sensor object.
+      /// \param[in] _sdf SDF Sensor parameters.
+      /// \return true if loading was successful
+      public: virtual bool Load(const sdf::Sensor &_sdf) override;
```

Comment:
nit: `virtual` is redundant if you have `override`. Could lead to compiler warnings in the future.

Reply:
Removed ``virtual``